build.gradle.kts

```diff
@@ -5,5 +5,20 @@ plugins {
 dependencies {
     // This merges all our individual kover results into the root project
     kover(project(":csaf-import"))
-    kover(project(":csaf-schema-codegen"))
-}
\ No newline at end of file
+}
+
+// Create and register ExecutionService which enforces serial execution of assigned tasks.
+abstract class SerialExecutionService : BuildService<BuildServiceParameters.None>
+val serialExecutionService =
+    gradle.sharedServices.registerIfAbsent("serialExecution", SerialExecutionService::class.java) {
+        this.maxParallelUsages.set(1)
+    }
+
+allprojects {
+    group = "io.github.csaf-sbom"
```

Comment:
this does not match the java package name `com.github.csaf.sbom` is this intentional?